tests/runtime/in_opentelemetry.c

```diff
@@ -552,6 +559,14 @@ static void flb_test_log_body()
     test_ctx_destroy(ctx);
 }
 
+static void flb_test_log_body() {
+    flb_test_logs_body_impl(NULL /* logs_body_key */, "{\"log\":\"{\\\"message\\\":\\\"test\\\"}\"}");
```

Comment:
These comments are superfluous and don't make the code easier to understand, please remove them and omit them in the future.

Comments are valuable and should be included but only when they clearly make it easier for whoever comes next to understand the context and reasoning behind a specific piece of code.

Reply:
Removed

(this is a [Go habit](https://github.com/uber-go/guide/blob/master/style.md#avoid-naked-parameters) of mine to help the reader know what argument a value like 1/true/NULL is used for)